model/godleyIcon.h

```diff
@@ -61,6 +59,8 @@ namespace minsky
       bb.update(*this); 
       selected=wasSelected;
     }
+  protected:
+     float m_sf=1;
```

Comment:
I meant use Item::m_sf. I thought maybe you didn't have access, because Item::m_sf might have been private, hence the comment about making it protected,  but I see it is actually public right now.

Reply:
"Overriding" a member variable like this is going to cause a lot of confusion going forward.